packages/rpc-spec/README.md

```diff
@@ -96,7 +96,7 @@ const rpcApi = createJsonRpcApi({
 rpcApi.foo('bar', { baz: 'bat' });
 
 // ...will produce an `RpcPlan` that:
-// -   Uses the following payload: { id: 1, jsonrpc: '2.0', method: 'foo', params: ['bar', { baz: 'bat' }] }.
+// -   Uses the following payload: { id: 1, jsonrpc: '2.0', method: 'foo', params: [{ baz: 'bat' }, 'bar'] }.
```

Comment:
This looked like an error to me. The transformer up there is supposed to `reverse()` the params